### PR TITLE
Add country rules for Poland and Singapore

### PR DESCRIPTION
**Pull request by camilavcoutinho**
Add country rules for Poland (POL) and Singapore(SGP)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+
+### Added
+
+- Specific rules for Poland (`POL`).
+- Specific rules for Poland (`SGP`).
```
Comment (batatavf2) [inline CHANGELOG.md:14]: I think you mean Singapore? 🤔

**Review by batatavf2 — CHANGES_REQUESTED** [review]
I think we need a PR for 3.x as well. Requested changes because i think there's a tiny mistake 😊 Besides that, LGTM ✨

File: CHANGELOG.md (hunk 1 above)
Comment (camilavcoutinho) [reply CHANGELOG.md]: My bad! Already fixed it.

**Comment by camilavcoutinho** [comment]
https://github.com/vtex/profile-form/pull/140, for 3.x

**Review by arthurepc — APPROVED** [review]

**Comment by arthurepc** [comment]
I'll merge this PR because @batatavf2 isn't available to remove the request changes

**Pull request merged**